lib/gremlin.js

```diff
@@ -19,15 +20,24 @@ var java = require('java');
 
 var Gremlin = module.exports = function (opts) {
 
-  if (opts && opts.java === java) {
+  // java.onJvmCreated is a property that holds a function that is executed only once when
+  // the JVM has been created. After creation, the property is made inaccessible.
+  var jvmCreated = !_.isFunction(java.onJvmCreated);
+
+  if (jvmCreated) {
+    dlog('Gremlin initialized after JVM already created. Java:', java);
+  }
+  else if (opts && opts.java) {
```

Comment:
Since we can now detect whether java has been initialized, why is it necessary to pass in the java object?